client/my-sites/pages/page-card-info/index.jsx

```diff
@@ -25,14 +54,15 @@ function PageCardInfo( { translate, moment, page, showTimestamp, isFront, isPost
 				</div>
 			}
 			<div>
-				{ showTimestamp &&
+				{ showTimestamp && (
+					renderFutureTimestamp() ||
```

Comment:
This should fix the linting errors. I was going to do something like this, but it would require returning an extra empty span tag.

``` js
const renderFutureTimestamp = function() {
	if ( page.status === 'future' ) {
		return (
			<PostRelativeTimeStatus
				post={ page }
				link={ contentLink.contentLinkURL }
				target={ contentLink.contentLinkTarget } />
		);
	}

	return (
		<span>
			<Gridicon icon="time" size={ iconSize } className="page-card-info__item-icon" />
			<span className="page-card-info__item-text">
				{ moment( page.modified ).fromNow() }
			</span>
		</span>
	);
};

return (
	<div className="page-card-info">
		{ siteUrl &&
			<div className="page-card-info__site-url">
				{ siteUrl }
			</div>
		}
		<div>
			{ showTimestamp && 
				<span className="page-card-info__item">
					{ renderFutureTimestamp() }
				</span>
			}
		</div>
	//...
);
```